Ginger/GingerCore/Drivers/Common/HTMLElementInfo.cs

```diff
@@ -88,6 +88,8 @@ public string Name
             set { mName = value; }
         }   // Developer Name
 
+        
+
         // Used for Lazy loading when possible
```

Comment:
need to remove this to extra rows so it will be removed from changes list compared to master